elki/src/main/java/ch/ethz/globis/pht/PhDimFilter.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2011-2015 ETH Zurich. All Rights Reserved.
+ *
+ * This software is the proprietary information of ETH Zurich.
+ * Use is subject to license terms.
```

Comment:
Here, the copying permission should be made explicit, as in the other ELKI files.
This is recommended by gnu.org ("copying permission statement"):
> To do so, attach the following notices to the program. It is safest to attach them to the start of each source file to most effectively state the exclusion of warranty; and each file should have at least the "copyright" line and a pointer to where the full notice is found.

"Proprietary information", "subject to license terms" does not state which license terms are applicable.